public/components/wz-search-bar/lib/suggest-handler.ts

```diff
@@ -108,7 +111,17 @@ export class SuggestHandler extends BaseHandler {
         return {suggestions};
       }
     } catch (error) {
-      return {error}
+      const options = {
+        context: `${SuggestHandler.name}.checkQuery`,
+        level: UI_LOGGER_LEVELS.ERROR,
+        severity: UI_ERROR_SEVERITIES.BUSINESS,
+        error: {
+          error: error,
+          message: error.message || error,
+          title: error.name || error,
+        },
+      };
+      getErrorOrchestrator().handleError(options);
```

Comment:
**thought**: Before the changes there was a `return {error}` and after that returns is removed, this changes the logic, and maybe that returns could be expected and managed in other places. We should check before removing it.